Fix error when building with no review services enabled

incompl opened the pull request:
### Screenshots

![Screen Shot 2022-11-22 at 1 28 17 PM](https://user-images.githubusercontent.com/211047/203424670-d88d134c-a219-4c64-8c5f-c4d9aa42cd00.png)

### Test Plan (Steps to test):

1. Log into the takeshape project at https://app.takeshape.io/project/06ccc3dc-a9da-4f5b-9142-5a104db52ee3/
1. Create a branch
1. On your branch, remove the authentication values for the `reviews-io` and `trustpilot` services
1. Pull down this PR and change `NEXT_PUBLIC_TAKESHAPE_API_URL` in `.env.local` to your branch's api url
1. Edit `src/config/reviews` and set `enableReviewsIo` to `false`
1. `npm run dev`
1. Check out the site, make sure nothing breaks
1. Try enabling the review services too and make sure they still work (for each you need to add the authentication value back in and edit `src/config/reviews` to enable

### Checklist:

- [x] Create a Test Plan
- [x] Changes Communicated (in commits or above)
- [x] Docs Updated
- [x] ~Storybook Updated~

mshick commented:
This general issue was already solved by migrating the Trustpilot config to the schema: https://github.com/takeshape/penny/blob/main/.takeshape/pattern/schema.json#L896

and making the component render conditional:

https://github.com/takeshape/penny/blob/main/src/features/ProductPage/ProductPage.tsx#L56-L66

The broader issue behind the related story is more something to be solved via making this is generator and implementing TakeShape transforms — being able to completely remove the services from the project without breaking type generation.

mshick reviewed (changes requested):
See note, we could just cleanup in this PR: remove the NEXT_PUBLIC env var for Trustpilot which is no longer needed, and update the docs. `ignoreErrors` might be a good add to the schema as well, but I don't think we need the component changes.

incompl commented:
> See note, we could just cleanup in this PR: remove the NEXT_PUBLIC env var for Trustpilot which is no longer needed, and update the docs. `ignoreErrors` might be a good add to the schema as well, but I don't think we need the component changes.

I've removed the Trustpilot NEXT_ENV variable and updated the README to indicate that you need to edit the endpoint URL with your business unit ID.

In my testing this `ignoreErrors` was required to run the project with reviews turned off, so yes lets leave it in.

The component changes are not as critical but they improve the page layout when reviews are disabled. For example they remove a huge blank area where reviews would otherwise have been.

mshick reviewed (approved):
Looks good!
